back_end/tweeter/lib/tweeter_web/router.ex

```diff
@@ -0,0 +1,46 @@
+defmodule TweeterWeb.Router do
+  use TweeterWeb, :router
+
+  pipeline :browser do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_live_flash
+    plug :put_root_layout, html: {TweeterWeb.Layouts, :root}
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", TweeterWeb do
+    pipe_through :browser
+    get "/", PageController, :home
```

Comment:
Remove this page controller, because our project doesnt need them, instead maybe just add API bare bone route path for `login` and `register` in two different `api` scope for authorized route path and otherwise, 

Refer the codebase i have shared earlier `realdealapi`